crates/sui-indexer-alt-framework/src/ingestion/client.rs

```diff
@@ -237,14 +238,14 @@ impl IngestionClient {
         self.metrics.total_ingested_inputs.inc_by(
             data.transactions
                 .iter()
-                .map(|tx| tx.input_objects.len() as u64)
+                .map(|tx| tx.input_objects(&data.object_set).count() as u64)
                 .sum(),
         );
 
         self.metrics.total_ingested_outputs.inc_by(
             data.transactions
                 .iter()
-                .map(|tx| tx.output_objects.len() as u64)
+                .map(|tx| tx.output_objects(&data.object_set).count() as u64)
                 .sum(),
```

Comment:
We may want to change this metric to just account for the object set as a whole.

Reply:
+1